### PR TITLE
[Content Copy for Prototyping] Copy of Site & Kit Content to use for Prototyping

### PR DESCRIPTION
**Pull request by BethanyG**
Per discussion [here](https://pyladies.slack.com/archives/C0170BUCTUL/p1716416701063729?thread_ts=1716399253.862839&cid=C0170BUCTUL), this is a first-pass of copying the main content of the site and kit to this new repo for prototyping purposes.  **This is not intended to be definitive nor complete, and is intended for use in prototyping different ideas and tech stacks only.**  

Best effort was made to convert HTML pages to their markdown content equivalents.  However, mistakes could very will have been made.  

For the `site/locations` folder, the chapter folders were retained, but the `index.html` pages were removed, since they pulled their information directly from the site `config.yml` file.  Chapter information that was in the site`config.yml` was moved to `site/locations/locations.yml`, with a `site/locations/locations.csv` version.  Site-specific config details were removed.

<details>
<summary>The current content tree</summary>

```bash
.
├── LICENSE
├── README.md
├── kit
│   ├── AUTHORS
│   ├── CONTRIBUTING.md
│   ├── README.md
│   ├── assets
│   │   ├── favicon.ico
│   │   └── images
│   │       ├── entypo_icon_name.png
│   │       ├── location_config
│   │       │   ├── complete.png
│   │       │   ├── coords.png
│   │       │   ├── email.png
│   │       │   ├── image.png
│   │       │   ├── meetup.png
│   │       │   ├── name.png
│   │       │   ├── twitter.png
│   │       │   ├── upcoming_meetups.png
│   │       │   └── website.png
│   │       ├── logos
│   │       │   ├── australia.png
│   │       │   ├── losangeles.jpeg
│   │       │   ├── pdx.png
│   │       │   ├── pyladies_wordmark_standard_black.png
│   │       │   └── pylady_geek_full_standard.png
│   │       ├── pylady_geek_greyblue_icon.png
│   │       └── pylady_geek_rainbow.png
│   ├── changelog.md
│   ├── faqs.md
│   ├── global
│   │   ├── award.md
│   │   ├── contact.md
│   │   ├── council.md
│   │   ├── index.md
│   │   └── pycon.md
│   ├── index.md
│   ├── member
│   │   ├── index.md
│   │   └── reporting.md
│   ├── organizer
│   │   ├── difficult
│   │   │   ├── burnout.md
│   │   │   ├── index.md
│   │   │   └── responding.md
│   │   ├── eventhosting
│   │   │   ├── advice.md
│   │   │   ├── events.md
│   │   │   ├── grant-proposals
│   │   │   │   └── sample-hackathon.md
│   │   │   ├── grants.md
│   │   │   ├── ideas.md
│   │   │   ├── index.md
│   │   │   ├── planning.md
│   │   │   ├── promotion.md
│   │   │   ├── resources
│   │   │   │   ├── djangozoom.md
│   │   │   │   └── index.md
│   │   │   ├── sponsors.md
│   │   │   └── talks
│   │   │       └── beginner.md
│   │   ├── general.md
│   │   ├── grants
│   │   │   └── sample-meetup-proposal.md
│   │   ├── index.md
│   │   ├── policies.md
│   │   └── website
│   │       ├── addresource.md
│   │       ├── blogpost.md
│   │       ├── devmainsite.md
│   │       ├── global.md
│   │       ├── index.md
│   │       ├── local.md
│   │       ├── location.md
│   │       ├── newicons.md
│   │       ├── simple.md
│   │       └── usingmynt.md
│   ├── overview.md
│   ├── policies
│   │   ├── coc.md
│   │   ├── index.md
│   │   ├── logo.md
│   │   └── trademark.md
│   ├── prospective
│   │   ├── checklist.md
│   │   ├── index.md
│   │   └── process.md
│   └── workshop_resources
│       ├── README.md
│       └── banner
│           └── README.md
└── site
    ├── archives
    ├── assets
    │   └── images
    │       ├── BPLlogo.png
    │       ├── Logo_Python.png
    │       ├── Pyladies_Panama.png
    │       ├── amchi_pyladies.png
    │       ├── atl_bnr_logo.png
    │       ├── blog
    │       │   ├── atxhack
    │       │   │   ├── ebay.png
    │       │   │   ├── first.png
    │       │   │   ├── numfocus.png
    │       │   │   ├── oreilly.png
    │       │   │   ├── partorgzs.png
    │       │   │   ├── psf32.png
    │       │   │   ├── rackspace.png
    │       │   │   ├── second.png
    │       │   │   └── third.png
    │       │   ├── austin_django_2014.jpeg
    │       │   ├── opw.png
    │       │   ├── pycon2013_lunch.jpg
    │       │   └── pycon_montreal.png
    │       ├── favicon.ico
    │       ├── mapmarker.png
    │       ├── netlify_logo.svg
    │       ├── pipinstall.png
    │       ├── pyladies-SCL.png
    │       ├── pyladies-SantiagoChile.png
    │       ├── pyladies-addis.png
    │       ├── pyladies-aracati.png
    │       ├── pyladies-asaba.png
    │       ├── pyladies-bcn.jpeg
    │       ├── pyladies-bh.png
    │       ├── pyladies-blumenau.png
    │       ├── pyladies-curitiba.png
    │       ├── pyladies-dublin.png
    │       ├── pyladies-equals-you.png
    │       ├── pyladies-geek-grey-blue.png
    │       ├── pyladies-geek-light-blue.png
    │       ├── pyladies-guatemala_city.jpg
    │       ├── pyladies-hermosillo.jpeg
    │       ├── pyladies-logo-navy.png
    │       ├── pyladies-oslo.png
    │       ├── pyladies-poa.png
    │       ├── pyladies-portoalegre.png
    │       ├── pyladies-remote.png
    │       ├── pyladies-rural.jpg
    │       ├── pyladies-seoul-logo.png
    │       ├── pyladies-silicion-logo.png
    │       ├── pyladies-sq-logo.png
    │       ├── pyladies-tunis.png
    │       ├── pyladies-tw-logo.png
    │       ├── pyladies-uy.jpeg
    │       ├── pyladies_amsterdam.jpeg
    │       ├── pyladies_ankara.png
    │       ├── pyladies_arequipa.jpg
    │       ├── pyladies_atx.jpeg
    │       ├── pyladies_australia.png
    │       ├── pyladies_belem.jpg
    │       ├── pyladies_berlin.png
    │       ├── pyladies_blr_logo.jpeg
    │       ├── pyladies_boston_logo_2_white.png
    │       ├── pyladies_brasil.png
    │       ├── pyladies_brasilia.png
    │       ├── pyladies_calgary.png
    │       ├── pyladies_campinas.png
    │       ├── pyladies_caxiasdosul.jpg
    │       ├── pyladies_cdmx.png
    │       ├── pyladies_charleston.png
    │       ├── pyladies_chennai.png
    │       ├── pyladies_chicago.png
    │       ├── pyladies_cochabamba.png
    │       ├── pyladies_cochabamba.png:Zone.Identifier
    │       ├── pyladies_colorado.png
    │       ├── pyladies_dc_atl_sea_tor.png
    │       ├── pyladies_df.png
    │       ├── pyladies_doboj.png
    │       ├── pyladies_ec.png
    │       ├── pyladies_edinburgh_logo2.png
    │       ├── pyladies_elalto.jpg
    │       ├── pyladies_enespanol.jpg
    │       ├── pyladies_es.png
    │       ├── pyladies_ghana.jpg
    │       ├── pyladies_grandrapids.png
    │       ├── pyladies_hamburg.png
    │       ├── pyladies_helsinki.png
    │       ├── pyladies_in.png
    │       ├── pyladies_kampala.png
    │       ├── pyladies_kzn.png
    │       ├── pyladies_la.jpeg
    │       ├── pyladies_lapaz.png
    │       ├── pyladies_logo.png
    │       ├── pyladies_london.png
    │       ├── pyladies_ma.png
    │       ├── pyladies_maceio.png
    │       ├── pyladies_madrid.png
    │       ├── pyladies_manaus.png
    │       ├── pyladies_melbourne.jpeg
    │       ├── pyladies_mtl.png
    │       ├── pyladies_nashville.jpeg
    │       ├── pyladies_natal.png
    │       ├── pyladies_ny.png
    │       ├── pyladies_pdx.png
    │       ├── pyladies_poland.png
    │       ├── pyladies_portoalegre.png
    │       ├── pyladies_pune.jpeg
    │       ├── pyladies_rdu.png
    │       ├── pyladies_recife.png
    │       ├── pyladies_rio.png
    │       ├── pyladies_salvador_logo.png
    │       ├── pyladies_santa_cruz.png
    │       ├── pyladies_saocarlos.png
    │       ├── pyladies_saoluis.png
    │       ├── pyladies_sd_brno.png
    │       ├── pyladies_sf_logo.png
    │       ├── pyladies_soflo.png
    │       ├── pyladies_sorocaba_logo.jpg
    │       ├── pyladies_spb.png
    │       ├── pyladies_sulmg.png
    │       ├── pyladies_teresina.png
    │       ├── pyladies_tnr.png
    │       ├── pyladies_tokyo.png
    │       ├── pyladies_vale.jpg
    │       ├── pyladiesimologo.png
    │       ├── pyladiesmu_logo.png.jpg
    │       ├── pylady_geek.png
    │       └── social
    │           ├── dribbble.png
    │           ├── facebook.png
    │           ├── github.png
    │           ├── instagram.png
    │           ├── lastfm.png
    │           ├── linkedin.png
    │           ├── slack.png
    │           ├── stack-overflow.png
    │           ├── steam.png
    │           └── twitter.png
    ├── blog
    │   ├── blog.md
    │   └── posts
    │       ├── 2011
    │       │   ├── 2011-04-19-first-python-ladies-night-recap.md
    │       │   ├── 2011-05-22-pyladies-intro-to-python-workshop-recap.md
    │       │   ├── 2011-06-02-python-ladies-night-3-on-june-2-2011.md
    │       │   ├── 2011-06-07-great-turnout-at-python-ladies-night-3.md
    │       │   ├── 2011-06-07-thank-you-python-software-foundation.md
    │       │   ├── 2011-06-24-recap-june-2011-hollywood-hackathon-at-border-stylo.md
    │       │   ├── 2011-06-30-call-for-sponsorship.md
    │       │   ├── 2011-07-12-django-workshop-and-hackathon-july-23rd.md
    │       │   ├── 2011-07-25-introduction-to-django-at-mahalo-july-23rd-pyladies-workshop.md
    │       │   ├── 2011-08-16-mid-august-news-bites.md
    │       │   ├── 2011-09-05-djangocon-sponsorship-program-results.md
    │       │   ├── 2011-09-05-pyladies-meetup-at-djangocon.md
    │       │   ├── 2011-09-06-djangsta-shirts-on-sale-at-djangocon.md
    │       │   ├── 2011-09-14-recap-pyladies-at-djangocon-2011.md
    │       │   ├── 2011-09-19-djangsta-shirts-extra-print-run.md
    │       │   ├── 2011-11-29-recaps-october-hackathon-at-idealab-november-beginners-workshop-at-usc.md
    │       │   ├── 2011-12-12-pyladies-at-pycon-2012.md
    │       │   └── 2011-12-15-call-for-applications-and-sponsors-for-pyladies-pycon-us-2012.md
    │       ├── 2012
    │       │   ├── 2012-01-22-get-the-most-out-of-your-conference-going-experience.md
    │       │   ├── 2012-02-24-pyladies-in-the-news.md
    │       │   ├── 2012-03-29-bay-area-girl-geek-dinner-at-yelp.md
    │       │   ├── 2012-04-12-Welcome-to-mynt.md
    │       │   ├── 2012-05-10-workshop-planning-session-python-ladies-night-2-recap.md
    │       │   ├── 2012-05-12-sf-study-group.md
    │       │   ├── 2012-05-17-jessica-mckellar-to-be-keynote-speaker-at-djangocon-europe.md
    │       │   ├── 2012-05-21-sf-pyladies-first-workshop-build-your-own-blog.md
    │       │   ├── 2012-06-06-python-it-up-in-rio.md
    │       │   ├── 2012-07-10-government-fellowship-opportunity.md
    │       │   ├── 2012-08-09-djangocon-in-dc.md
    │       │   └── 2012-08-25-pycon-proposal-brainstorm-session-aug-27th.md
    │       ├── 2013
    │       │   ├── 2013-01-13-pyladies-speaking-at-pycon.md
    │       │   ├── 2013-01-28-PyLadies-Lunch-at-PyCon.md
    │       │   ├── 2013-02-23-Why-PyLadies.md
    │       │   ├── 2013-02-26-how-I-learned-to-love-rebase.md
    │       │   ├── 2013-03-06-Get-Your-Mac-Ready-for-Python-Programming.md
    │       │   ├── 2013-03-07-PyLadies-at-PyCon-2013.md
    │       │   ├── 2013-04-04-computer-science-outreach-magic.md
    │       │   ├── 2013-04-10-Code-of-Conduct.md
    │       │   ├── 2013-04-10-intro-python-april-6-recap.md
    │       │   ├── 2013-04-19-libraries-resources.md
    │       │   ├── 2013-04-19-newsletters-resources.md
    │       │   ├── 2013-04-19-speaking-resources.md
    │       │   ├── 2013-04-19-tools-resources.md
    │       │   ├── 2013-05-07-irc-resources.md
    │       │   ├── 2013-05-07-user-group-resources.md
    │       │   ├── 2013-07-16-pycon-call-for-proposals.md
    │       │   ├── 2013-08-17-getting-more-women-to-submit-talk-proposals.md
    │       │   ├── 2013-08-18-pycon-2014-cfp-brainstorm.md
    │       │   ├── 2013-10-04-lightning-talks.md
    │       │   └── 2013-10-07-fundraising-for-pycon-2014.md
    │       ├── 2014
    │       │   ├── 2014-01-30-pycon-tutorials.md
    │       │   ├── 2014-03-03-opw.md
    │       │   ├── 2014-03-05-opw-office-hours.md
    │       │   ├── 2014-03-10-pyladies-pycon-2014-lunch.md
    │       │   ├── 2014-03-27-pyladies-events-at-pycon2014.md
    │       │   ├── 2014-04-18-pip-install-pyladies.md
    │       │   ├── 2014-04-26-Attend-Google-IO-with-PyLadies.md
    │       │   ├── 2014-07-27-call-for-blog-posts.md
    │       │   ├── 2014-07-29-tutorial-resources.md
    │       │   ├── 2014-08-04-CfP-PyCon-2015.md
    │       │   ├── 2014-08-05-the-first-django-girls-happened.md
    │       │   ├── 2014-08-06-python-is-for-everyone.md
    │       │   ├── 2014-08-24-austin-django-workshop-recap.md
    │       │   ├── 2014-08-25-community-conduct-and-compassion-resources.md
    │       │   ├── 2014-08-25-on-compassion-community-and-caring.md
    │       │   ├── 2014-08-26-speaking-pycon.md
    │       │   ├── 2014-09-01-austin-ladies-hack.md
    │       │   ├── 2014-09-14-community-workshop-resources.md
    │       │   ├── 2014-11-02-fundraising-for-pycon-2015.md
    │       │   └── 2014-12-01-Donate-to-PyLadies-at-PyCon.md
    │       ├── 2015
    │       │   ├── 2015-02-01-pyladies-pytn.md
    │       │   ├── 2015-03-23-pyladies-at-pycon.md
    │       │   └── 2015-05-03-django-resources.md
    │       ├── 2016
    │       │   └── 2016-08-01-europython-recap.md
    │       ├── 2019
    │       │   ├── 2019-08-15-important-pyladies-update.md
    │       │   ├── 2019-11-04-help-us-with-pyladies-project-and-council-selection.md
    │       │   └── 2019-12-18-pyladies-global-council-process-vote.md
    │       ├── 2020
    │       │   ├── 2020-07-01-announcement-for-registering-pyladies-members.md
    │       │   └── 2020-09-21-inaugural-pyladies-council.md
    │       ├── 2021
    │       │   ├── 2021-06-20-global-council-announcement.md
    │       │   └── PyLadies_Test_Post.md
    │       ├── 2023
    │       │   ├── 2023-06-22-global-conference-announcement.md
    │       │   ├── 2023-07-28-cfp-launch-announcement.md
    │       │   ├── 2023-08-17-cfp-workshop-announcement.md
    │       │   ├── 2023-09-03-cfp-outreach-kit-en.md
    │       │   ├── 2023-09-05-cfp-outreach-kit-es.md
    │       │   ├── 2023-09-05-cfp-outreach-kit-fr.md
    │       │   ├── 2023-09-05-cfp-outreach-kit-pt.md
    │       │   ├── 2023-09-08-cfp-workshop-thanks.md
    │       │   ├── 2023-09-10-cfp-extended.md
    │       │   ├── 2023-09-15-cfp-outreach-kit-zh-simplified.md
    │       │   ├── 2023-09-15-cfp-outreach-kit-zh-traditional.md
    │       │   ├── 2023-10-02-sponsorship-announcement.md
    │       │   ├── 2023-10-03-cfp-closed.md
    │       │   └── 2023-11-03-chapters-video-request.md
    │       └── 2024
    │           ├── 2024-01-22-outstanding-award.md
    │           └── 2024-03-06-pyladies-at-pycon-us.md
    ├── feed.xml
    └── pages
        ├── about
        │   └── about.md
        ├── archives
        │   └── archives.md
        ├── codeofconduct
        │   └── codeofconduct.md
        ├── home.md
        ├── locations
        │   ├── NG
        │   ├── amsterdam
        │   ├── ankara
        │   ├── atl
        │   ├── atx
        │   ├── bbsr
        │   ├── berlin
        │   ├── blr
        │   ├── boston
        │   ├── cle
        │   ├── cochabamba
        │   ├── cz
        │   ├── dallas
        │   ├── dc
        │   ├── dublin
        │   ├── edinburgh
        │   ├── elalto
        │   ├── hamburg
        │   ├── helsinki
        │   ├── kampala
        │   ├── la
        │   ├── locations.csv
        │   ├── locations.yml
        │   ├── london
        │   ├── lp
        │   ├── mavs
        │   ├── mtl
        │   ├── nash
        │   ├── paris
        │   ├── pdx
        │   ├── pl
        │   ├── portoalegre
        │   ├── rdu
        │   ├── saltlake
        │   ├── sd
        │   ├── sea
        │   ├── seoul
        │   ├── sf
        │   ├── sg
        │   ├── stockholm
        │   ├── tnr
        │   ├── toronto
        │   ├── tunis
        │   ├── vancouver
        │   └── vienna
        ├── resources
        │   └── resources.md
        ├── sponsor
        │   └── sponsor.md
        └── tags
```

</details>

**Maintainers/reviewers:**  Please feel free to edit as needed, since this was a best guess.

### CODE REVIEW

**Review by willingc — COMMENTED** [review]
Thanks Bethany. Overall, it looks good. I would remove all of the `.DS_Store` files. Probably most easily done by adding a `.gitignore` file and listing it in the file.

**Comment by BethanyG** [comment]
> I would remove all of the .DS_Store files. Probably most easily done by adding a .gitignore file and listing it in the file.

Gah!  I thought I'd done that, but clearly I hadn't.  🤦🏽‍♀️   Will update.  😄

File: site/feed.xml
Comment (willingc) [inline site/feed.xml:1]: We can remove this file since it will autogenerate from any of the frameworks chosen.

**Review by willingc — COMMENTED** [review]
@BethanyG Let's double check the images pulled over from the kit. They seem unable to render and the `favicon.ico` is empty. Maybe something wonky happened with the copy.

File: site/feed.xml
Comment (willingc) [reply site/feed.xml:1]: Who knows maybe it was some latency or connectivity glitch. They look good now. Thanks.

**Review by willingc — APPROVED** [review]
Thanks @BethanyG. :ship:

**Comment by BethanyG** [comment]
@econchick  & @trallard -- any comments/changes/concerns before we merge this?

**Review by trallard — APPROVED** [review]
I only spotted a few images that should perhaps be removed and a couple of straggling `.DS_Store` still buried in here.

I think this is a great foundation to start prototyping from so I would suggest merging soon and make changes iteratively and as needed.

**Review by trallard — APPROVED** [review]
I only spotted a few images that should perhaps be removed and a couple of straggling `.DS_Store` still buried in here.

I think this is a great foundation to start prototyping from so I would suggest merging soon and make changes iteratively and as needed.

**Comment by willingc** [comment]
I think `**/.DS_Store` will get the files in the subdirectories.

**Comment by BethanyG** [comment]
> I think `**/.DS_Store` will ignore the files in the subdirectories.

UGH.  Thank you, @willingc.  Sorry this has been such a mess.  I will do a final cleanup, and add that to the `.gitignore`.

File: kit/assets/images/entypo_icon_name.png
[the remark this answers is not in the document]
Comment (trallard) [reply kit/assets/images/entypo_icon_name.png:1]: Seems my other comment went missing but the images in `kit/assets/images/<something something>` are the ones I am referring to, they seem like some screenshots vs real icons which is what I'd expect.

[the remark this answers is not in the document]
Comment (BethanyG) [reply kit/assets/images/entypo_icon_name.png:1]: My comments appear to have vanished as well. Those images are all intended to be screenshots.  Not sure where we put them organizationally, but they are part of the `How-To` [here](https://kit.pyladies.com/en/latest/organizer/website/global.html#adding-your-location).

They were originally under https://github.com/pyladies/pyladies-kit/tree/master/docs/_static/images.

**Comment by BethanyG** [comment]
Ok.  I think we can move around the screengrabs later.  Right now, I am going to merge, and if I've still got a `.DS_Store` in there, I will nuke it later.  😄

**Pull request merged**